app/assets/stylesheets/base/_colors.scss

```diff
@@ -8,7 +8,7 @@ $header-text-color: #fff;
 $link-color: #02779E;
 $subtle-secondary-text-color: #b5839b;
 $overlay-color: rgba(225, 205, 215, 0.2);
-$field-color: rgba(0, 0, 0, 0.2);
+$field-color: rgba(109, 8, 57, 0.4);
```

Comment:
I think for now, let's keep in black. And then we can revisit the colours in a later story?